tests/Instantiator/ConstructInstantiatorTest.php

```diff
@@ -3,13 +3,14 @@
 
 namespace Linio\Component\Input\Instantiator;
 
+use Linio\Component\Input\Constraint\Enum;
+
 class ConstructInstantiatorTest extends \PHPUnit_Framework_TestCase
 {
     public function testIsCreatingInstances()
     {
         $instantiator = new ConstructInstantiator();
-        $instance = $instantiator->instantiate('ErrorException', ['foobar']);
-        $this->assertInstanceOf('ErrorException', $instance);
-        $this->assertEquals(new \ErrorException('foobar'), $instance);
+        $instance = $instantiator->instantiate(Enum::class, ['foo' => [1, 2], 'bar' => 'message']);
+        $this->assertEquals(new Enum([1, 2], 'message'), $instance);
```

Comment:
I'm guessing you didn't READ this then right? If you did, a new test would have been created to cover this issue. Without a new test who's to say this test wont be changed in the future thus allowing this big to be reintroduced?